tslint-rules/BUILD.bazel

```diff
@@ -1,13 +1,20 @@
-load("@npm//@bazel/typescript:index.bzl", "ts_library")
+load("//tools:defaults.bzl", "pkg_npm", "ts_library")
 
 ts_library(
     name = "tslint-rules",
     srcs = glob(["*.ts"]),
     module_name = "@angular/dev-infra-private/tslint-rules",
     visibility = ["//dev-infra:__subpackages__"],
     deps = [
-        "@npm//tslib",
         "@npm//tslint",
         "@npm//typescript",
     ],
 )
+
+pkg_npm(
+    name = "npm_package",
+    build_package_json_from_template = True,
```

Comment:
Why do the individual sub-folders need a `package.json` at all? Wouldn't we just use the project root one this time? with proper separation of `devDependencies` and `dependencies`.

Reply:
The idea was that we actually are creating 3 distinct packages, so the deps that are listed for each should be managed independently of eachother.

Reply:
how would that reflect in NPM? since we publish the dev-infra package to NPM as a single package (is that still the plan?), the nested `package.json` dependencies don't really have an effect.

I assume it would be more convenient (and the standard way) to have a single root package.json (for intelli-sense) that we need to maintain? WDYT

Reply:
I was under the impression/understanding we were going to be splitting into three packages: `ng-dev`, `tslint-rules` and `bazel`